environments/base/default-config.libsonnet

```diff
@@ -97,6 +97,15 @@
     },
   },
 
+  storeCache: {
```

Comment:
We need to update the operator code also:
https://github.com/observatorium/configuration/blob/master/api/v1alpha1/observatorium_types.go (and generate CRD)
https://github.com/observatorium/configuration/blob/master/environments/operator/operator-config.libsonnet
https://github.com/observatorium/configuration/blob/master/example/main.jsonnet

Reply:
@rollandf I've tried to update operator sources, could you please have look? 
Also in the tests, I've seen some errors around queried, do you know anything about them? or is it just a glitch?

Reply:
`Error from server (NotFound): deployments.apps "observatorium-xyz-thanos-query" not found`

Reply:
```
E0407 10:23:01.722528       1 resource.go:177] Sync "observatorium/observatorium-xyz" failed: failed to render: failed to evaluate: RUNTIME ERROR: must provide exporter image
	components/memcached.libsonnet:12:21-56	object <anonymous>
	components/memcached.libsonnet:82:33-56	thunk from <thunk <exporter> from <object <anonymous>>>
	vendor/ksonnet/ksonnet.beta.4/k8s.libsonnet:22402:51-56	
	vendor/ksonnet/ksonnet.beta.4/k8s.libsonnet:22422:46-51	object <anonymous>

```